Workbooks/Azure Active Directory Provisioning/AAD cloud sync Insights (Preview)/AAD cloud sync Insights.workbook

```diff
@@ -573,7 +573,7 @@
       "content": {
         "version": "NotebookGroup/1.0",
         "groupType": "template",
-        "loadFromTemplateId": "Community-Workbooks/Microsoft Entra ID Provisioning/AAD cloud sync details",
+        "loadFromTemplateId": "Community-Workbooks/Azure Active Directory Provisioning/AAD cloud sync details",
```

Comment:
This is an id to the linked workbook, we did not change the id or location of that workbook so this id is invalid. Reverting back to the old one until we change the path for all workbooks